<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="1" column="0">
    <widget class="QPushButton" name="aboutPushButton">
     <property name="text">
      <string>&amp;About</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="restartPushButton">
     <property name="text">
      <string>Re&amp;start</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QStackedWidget" name="centralStackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="NotLoggedInPage">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="notLoggedInVerticalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayoutInput">
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditPass">
             <property name="font">
              <font>
               <family>Lucida Sans Typewriter</family>
              </font>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelAccount">
             <property name="text">
              <string>Account</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditAccount"/>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comboBoxMAC">
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="toolTip">
              <string>To use a customized MAC address, clear the text fleld and type one using dash or colon as separator</string>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelPass">
             <property name="text">
              <string>Password</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0" rowspan="2">
            <widget class="QLabel" name="labelMAC">
             <property name="text">
              <string>MAC Address</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="checkboxesGridLayout">
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBoxRemember">
             <property name="text">
              <string>Remember credential</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkBoxAutoLogin">
             <property name="text">
              <string>Auto login</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBoxNotShowWelcome">
             <property name="text">
              <string>Don't show welcome page</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="checkBoxHideLoginWindow">
             <property name="text">
              <string>Hide this window</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="loginButton">
           <property name="text">
            <string>Login</string>
           </property>
           <property name="description">
            <string>Connect to auth server; access the Internet</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="LoggedInPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="loggedInGridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="5" column="0" colspan="2">
          <widget class="QCommandLinkButton" name="browserButton">
           <property name="text">
            <string>Launch &amp;browser</string>
           </property>
           <property name="description">
            <string>Visit ip.jlu.edu.cn using default browser</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="alreadyLoggedInLabel">
           <property name="text">
            <string>Logged in. You may close this window and reopen it later using tray icon.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="ipLabel">
           <property name="text">
            <string notr="true">0.0.0.0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="uptimeLabel">
           <property name="text">
            <string notr="true">0:00:00</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="ipHintLabel">
           <property name="text">
            <string>IP Address</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="loggedInUsernameLabel">
           <property name="text">
            <string>Account</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="uptimeHintLabel">
           <property name="text">
            <string>Uptime</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="loggedInUsernameHintLabel">
           <property name="text">
            <string>Logged in as</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QCommandLinkButton" name="logoutButton">
           <property name="text">
            <string>&amp;Logout</string>
           </property>
           <property name="description">
            <string>Disconnect from the auth server</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditAccount</tabstop>
  <tabstop>lineEditPass</tabstop>
  <tabstop>comboBoxMAC</tabstop>
  <tabstop>checkBoxRemember</tabstop>
  <tabstop>checkBoxAutoLogin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
